modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -68,4 +68,42 @@
       evss_response { 'something' }
     end
   end
+
+  factory :bsg_response, class: OpenStruct do
+    bnft_claim_dto { create(:bnft_claim_dto).to_h }
+  end
+
+  factory :bnft_claim_dto, class: OpenStruct do
+    bnft_claim_id { '600333346' }
+    bnft_claim_type_cd { '020CLMINC' }
+    bnft_claim_type_label { 'Compensation' }
+    bnft_claim_type_nm { 'Claim for Increase' }
+    bnft_claim_user_display { 'YES' }
+    claim_jrsdtn_lctn_id { '123725' }
+    claim_rcvd_dt { '2022-09-23' }
+    cp_claim_end_prdct_type_cd { '021' }
+    jrn_dt { '2022-09-26T11:43:04' }
+    jrn_lctn_id { '283' }
+    jrn_obj_id { 'cd_clm_lc_status_pkg.do_create' }
+    jrn_status_type_cd { 'U' }
+    jrn_user_id { 'VBMSSYSACCT' }
+    payee_type_cd { '00' }
+    payee_type_nm { 'Veteran' }
+    pgm_type_cd { 'CPL' }
+    pgm_type_nm { 'Compensation-Pension Live' }
+    ptcpnt_clmant_id { '600043201' }
+    ptcpnt_clmant_nm { 'ELLIS TAMARA' }
+    ptcpnt_mail_addrs_id { '14981239' }
+    ptcpnt_pymt_addrs_id { '14906550' }
+    ptcpnt_vet_id { '600043201' }
+    ptcpnt_vsr_id { '600831201' }
+    station_of_jurisdiction { '499' }
+    status_type_cd { 'RFD' }
+    status_type_nm { 'Ready for Decision' }
+    submtr_applcn_type_cd { 'VBMS' }
+    submtr_role_type_cd { 'VBA' }
+    svc_type_cd { 'CP' }
+    termnl_digit_nbr { '15' }
+    filed5103_waiver_ind { 'N' }
+  end
```

Comment:
nitpick: We could use [faker](https://github.com/faker-ruby/faker) to generate random data... would be better to actually test the logic itself and not assume we're going to get a hard coded value, plus it's already in the gemfile. That being said, I think we're using cassettes for all the responses anyway. Unless we're manipulating data before sending it in I guess it's probably irrelevant?